docs/user/basics/documents-creator.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Documents Creator
+description: Create user-friendly documents
+lang: en
+---
+
+# Documents Creator
+
+The Documents creator converts your documentation data into user-friendly documents. This allows you to reach a diverse audience - from tech-savvy stakeholders to the less digitally fluent decision-makers.
+
+The documents can serve as a documentation basis for audits, conformity checks and customer reporting. Documents are available in the main navigation bar.
+
+## Rights
+
+You should check the [Rights](../../admin/rights-and-permissions.md) and alter them to fit your needs. It is also possible to set `All available Rights (inlcuding future updates)`.
+
+## Design your documents
+
+One size doesn't fit all, especially in the world of IT Asset Management. Different stakeholders have different needs. Customize your document with different chapters and sections and decide what elements of your IT asset data take center stage.
+
+### Global documents
+
+Global documents can be viewed by any tenant.
+
+### Tenant documents
+
+Tenant documents are only available for the current tenant.
+
+### Document settings
+
+Each document needs to have a Document title and can have a description.
+
+### Chapters
+
+There are default Chapter `Title page` and `Table of content`. Also there is a add button to add `custom chapter` for your needs.
+
+#### Sections
+
+To fill the Chapter with data you have to Option to add `Object details` section, `Text` section, `Headline` section and a `Custom Table`. The `Headline` is added by default. In the `Text` section you can use supported [Markdown](https://www.markdownguide.org/basic-syntax/) to build high quality documents. Sections can be reordered or deleted.
```

Comment:
to Option to add -> sounds weird.

Might be a good idea to make an unordered list from the section selection.